src/Application.php

```diff
@@ -694,4 +628,18 @@ private function swallowDeprecationNotices()
 
         $previous = set_error_handler($handler);
     }
+
+    /**
+     * @var \Exception|\Throwable $exception
```

Comment:
`Exception` cannot be imported, as it would conflict with the `Zend\Expressive\Exception` namespace. I'll import `Throwable`, however.